include/oneapi/dpl/pstl/tuple_impl.h

```diff
@@ -29,12 +29,39 @@ namespace dpl
 {
 namespace __internal
 {
-
-//custom tuple utilities
-
 template <typename... T>
 struct tuple;
+} // namespace __internal
+} // namespace dpl
+} // namespace oneapi
 
+namespace std
```

Comment:
Could you please clarify why we need to inject `tuple_element`, `tuple_size` to `std`?

Reply:
It's written in description. You need to say somehow that you type is tuple-like. Without that structured binding doesn't work.

Reply:
Sorry, didn't recognize that it's the same thing.